tests/api/test_account.py

```diff
@@ -297,6 +297,52 @@ def test_user_query_anonymous_user(api_client):
     assert_no_permission(response)
 
 
+def test_user_query_permission_manage_users_get_customer(
+    staff_api_client, customer_user, permission_manage_users
+):
+    staff_api_client.user.user_permissions.add(permission_manage_users)
+    customer_id = graphene.Node.to_global_id("User", customer_user.pk)
+    variables = {"id": customer_id}
+    response = staff_api_client.post_graphql(USER_QUERY, variables)
```

Comment:
I think it would be best if we passed the permission directly to `post_graphql` so we also check it raises permission denied if the permission are missing for each case (get customer with X perm, get staff with X perm).

Same for below.